packages/components/organisms/f-checkout/src/services/addressService.js

```diff
@@ -70,6 +72,32 @@ function getAddressClosestToPostcode (postcode, addresses) {
     return address;
 }
 
+const isAddressInLocalStorage = () => {
+    if (window.localStorage) {
+        const address = window.localStorage.getItem(storedLocationKey) ? JSON.parse(window.localStorage.getItem(storedLocationKey)) : null;
+        return !!(address && (address.PostalCode || address.ZipCode));
```

Comment:
```suggestion
        return !!(address?.PostalCode || address?.ZipCode);
```

Reply:
Unfortunately we can't use optional chaining with Storybook right now. I believe we're either waiting for the next version of Storybook or the next version of Vue.

Reply:
I would probably even leave it like this: 
```
return address?.PostalCode || address?.ZipCode;
```
as for me, it's much more readable

Reply:
oh I see!